testdata/project-v3-config/channels/stable

```diff
@@ -0,0 +1,3 @@
+# Versions for the stable channel
+manifests:
+- version: 0.0.1
```

Comment:
Why we are doing this scaffold now?

Reply:
That is part of the declarative pattern, it modifies `*_types.go` and `*_controller.go` and creates two placeholder files.